src/main/java/com/artipie/npm/proxy/model/NpmPackage.java

```diff
@@ -79,4 +93,12 @@ public String content() {
     public String lastModified() {
         return this.modified;
     }
+
+    /**
+     * Get last updated date.
+     * @return Last updated date
+     */
+    public OffsetDateTime lastUpdated() {
```

Comment:
@brastak this is not a good name for this method because it's meaning doesn't differ from `lastModified`. How about `lastRefreshed`?

Reply:
@Vatavuk I will rename it to `lastModifed` and `lastRefreshed`. `Last-Modified` property comes from remote repository and it's better to leave it "as-is". And `lastRefreshed` is our attribute.